viewer/viewer.js

```diff
@@ -7702,14 +7701,15 @@ function continueHuntSkipSession (hunt, huntId, session, sessionId, searchedSess
   updateHuntStats(hunt, huntId, session, searchedSessions, cb);
 }
 
+let searchingFailedSessions = false; // TODO ECR - a better place to put this? internals?
```

Comment:
@awick thoughts on the best place to put this?

Reply:
Ya shouldn't be a global incase somehow multiple hunts are running or some other badness. I would look at having buildHuntOptions put it inside of options if that would work.

Reply:
👍 I put it on the hunt options object